config/initializers/rollbar.rb

```diff
@@ -59,7 +59,9 @@
   config.js_options = {
     accessToken: ENV.fetch('ROLLBAR_CLIENT_ACCESS_TOKEN'),
     captureUncaught: true,
-    hostWhiteList: ['staging.coinfi.com', 'www.coinfi.com', 'sale.coinfi.com'],
+    captureUnhandledRejections: true,
+    logLevel: 'error',
+    hostWhiteList: ['coinfi.com', 'herokuapp.com'],
```

Comment:
Does an entry in `hostWhiteList` include all subdomains?

If not, it would be useful to be able to test this on the develop and staging subdomains as well, and the herokuapp.com domain would be useless unfortunately :(

Reply:
Yeah according to the [documentation](https://docs.rollbar.com/docs/reduce-noisy-javascript-errors#section-whitelist-specific-domains) it treats it like regex:

>When hostWhiteList has been configured, Rollbar will enumerate over the stack frames in each error. At least one of the frames must contain a filename that contains at least one of the strings in this configuration option. The items in the array are compiled into a regex that is used to compare against the filenames.

Reply:
Awesomesauce, thanks!